docs/ssm-deployment-guide.md

```diff
@@ -213,9 +213,12 @@ Enter into ${SMART_HOME} directory for running SSM.
 
    `./bin/start-ssm.sh`
 
+   `version` can be used to show specific version information for SSM.
```

Comment:
Here, the options are for start-ssm.sh use. So you should move this statement after "Enter into ${SMART_HOME} directory for running SSM." which is in above. In addition, modify to `ssm version` can be used ...